### PR TITLE
Merge 2.0.0 changes to master

### PR DESCRIPTION
**Pull request by littleaj**
This brings the branches back together after the fork for 2.0.0.

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
 // Project properties
-version=2.0.0-BETA
+version=2.0.0
```
Comment (grlima) [inline gradle.properties]: There are missing STABLE release commits in this PR. Stable branch is at version 2.0.1, for example.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,10 @@
 - Fixed PageView telemetry data not being reported. 
 - Fixed Issue #526 (NPE in MapUtil.copy())
 
+## Version 2.0.0
+
+- Fix #506 Null Reference Check causing Null Pointer Exception in `TelemetryCorrelationUtils.java`
+
```
Comment (grlima) [inline CHANGELOG.md]: There should be more items in this changelog for 2.0.0 - i believe this PR is missing the last commits from Stable branch

File: gradle.properties (hunk 1 above)
Comment (dhaval24) [reply gradle.properties]: @littleaj did you pulled the latest stable before merging locally?

**Comment by dhaval24** [comment]
Also FYI don't delete the stable branch. We might need if we want to do a hot fix.

**Comment by littleaj** [comment]
@grlima @dhaval24 Fixed. I needed to update my local 2.0.0-STABLE apparently.

Also, Since we have a tag on the commit we released, we can delete the branch; but I'll leave it for convenience. If we ever need to do this again, we can just merge the _version_ branch (e.g. 2.0.0-STABLE) straight into master after the release is tagged. We won't lose the commit as long as something is pointing to it. See this post: https://stackoverflow.com/questions/33283350/what-happen-to-git-tags-pointing-to-a-removed-commit

File: gradle.properties (hunk 1 above)
Comment (littleaj) [reply gradle.properties]: @dhaval24 nope :) fixed.

**Comment by lhotari** [comment]
@littleaj please review #582 since that's a PR for 2.0.0-STABLE branch.

**Comment by dhaval24** [comment]
@lhotari thanks for pointing out. Yes we should review this before merge.

**Comment by littleaj** [comment]
@grlima I think all the issues have been addressed.

**Pull request merged**